apis/r/src/Makevars.in

```diff
@@ -3,24 +3,21 @@ CXX_STD = CXX17
 ## We need the TileDB Headers, and for macOS aka Darwin need to set minimum version 10.14 for macOS
 PKG_CPPFLAGS = -I. -I../inst/include/ @tiledb_include@ @cxx17_macos@ -Iinclude
```

Comment:
We need separate `@tiledb_include@`/`@tiledb_libs@`, and `@tiledbsoma_include@`/`@tiledbsoma_libs@`

Reply:
You are welcome to fork and extend.  The PR is simple and works.